addons/medical_damage/stringtable.xml

```diff
@@ -705,6 +705,14 @@
             <Spanish>La probabilidad de que una persona caiga inconsciente cuando su dolor está por encima del umbral al haber recibido daño.</Spanish>
             <French>La probabilité pour qu'une personne perde connaissance lorsque la douleur ressentie est supérieure à son seuil de tolérance.</French>
         </Key>
+        <Key ID="STR_ACE_Medical_Damage_PainUnconsciousThreshold_DisplayName">
+            <English>Pain Unconscious Threshold</English>
+            <Russian>Порог боли для потери сознания</Russian>
+        </Key>
+        <Key ID="STR_ACE_Medical_Damage_PainUnconsciousThreshold_Description">
+            <English>Sets the amount of pain from the received wound, at which a unit can lose consciousness.</English>
```

Comment:
```suggestion
            <English>Sets the threshold for severe pain, above which unit can lose consciouness when receiving a wound.</English>
```

More readable IMO and lets user know what they're looking for when changing the setting.